inginious/agent/docker_agent/__init__.py

```diff
@@ -62,9 +63,9 @@ class DockerRunningStudentContainer:
     assigned_external_ports: List[int]
 
 
-
 class DockerAgent(Agent):
-    def __init__(self, context, backend_addr, friendly_name, concurrency, tasks_fs: FileSystemProvider, address_host=None, external_ports=None, tmp_dir="./agent_tmp", runtimes=None, ssh_allowed=False):
+    def __init__(self, context, backend_addr, friendly_name, concurrency, tasks_fs: FileSystemProvider,
```

Comment:
It would be great to avoid changing the formatting in the same commits as doing important changes. This makes the review more difficult and potentially makes bug catching, rebase and backport cherry-picking more tricky.

Reply:
Very well!